TCMSApp/src/client/app/layout/sidebar.service.js

```diff
@@ -0,0 +1,46 @@
+(function () {
```

Comment:
styleguide and experts teach us too write as follows: 

```js    
function sidebarFactory(moment) {
        return {
            findProjectsNames: addProject,
            addProject: addProject
        }
    }
```
and declare functions at the bottom